.github/workflows/lint-php.yml

```diff
@@ -13,29 +13,40 @@ on:
       - master
       - stable*
 
+permissions:
+  contents: read
+
+concurrency:
+  group: lint-php-${{ github.head_ref || github.run_id }}
+  cancel-in-progress: true
+
 jobs:
   php-lint:
     runs-on: ubuntu-latest
     strategy:
       matrix:
-        php-versions: ["7.4", "8.0"]
+        php-versions: [ "8.0", "8.1", "8.2" ]
```

Comment:
This should be changed back, no? 
https://docs.nextcloud.com/server/25/admin_manual/installation/system_requirements.html lists 7.4, 8.0, 8.1.